src/Processors/Executors/ExecutionThreadContext.h

```diff
@@ -43,6 +43,9 @@ class ExecutionThreadContext
     const bool profile_processors;
     const bool trace_processors;
 
+    constexpr static size_t max_consequintely_scheduled_local_tasks = 128;
```

Comment:
What's `consequintely`? I guess it's a typo, but I can't come up with the fixed word

Reply:
Yes, not a very good name.
Well, I can change it to just `max_scheduled_local_tasks` and write some comment.